src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-button-strategy.ts

```diff
@@ -74,15 +75,29 @@ export default class PaypalCommerceButtonStrategy implements CheckoutButtonStrat
     }
 
     private _getParamsScript(initializationData: PaypalCommerceInitializationData, cart: Cart): PaypalCommerceScriptParams {
-        const { clientId, intent, isPayPalCreditAvailable, merchantId, attributionId } = initializationData;
+        const {
+            clientId,
+            intent,
+            isPayPalCreditAvailable,
+            merchantId,
+            attributionId,
+            availableAlternativePaymentMethods = [],
+            enabledAlternativePaymentMethods = [],
+        } = initializationData;
 
-        const disableFunding: DisableFundingType = [ 'card' ];
-        const enableFunding: EnableFundingType = [];
+        const disableFunding: FundingType = [ 'card' ];
+        const enableFunding: FundingType = enabledAlternativePaymentMethods.slice(0);
```

Comment:
```suggestion
        const enableFunding: FundingType = [...enabledAlternativePaymentMethods]
```

Reply:
@mjhmatt checkout-sdk uses old version of TSlib.
You'll get this error
This syntax requires an imported helper named '__spreadArray' which does not exist in 'tslib'. Consider upgrading your version of 'tslib'